edgelet/iotedge/src/config/import/mod.rs

```diff
@@ -124,19 +122,12 @@ fn execute_inner(
     };
 
     let old_config: old_config::Config = {
-        let mut old_config = Config::default();
-
-        old_config
-            .merge(YamlFileSource::String(old_config::DEFAULTS.into()))
-            .expect("config is not frozen");
-
-        // We use YamlFileSource::String to load the file rather than YamlFileSource::File
-        // because config::ConfigError makes it harder to recognize an error from a missing file.
-        old_config
-            .merge(YamlFileSource::String(old_config_contents.into()))
-            .expect("config is not frozen");
+        let old_config = config::ConfigBuilder::<config::builder::DefaultState>::default()
+            .add_source(YamlFileSource::String(old_config::DEFAULTS.into()))
+            .add_source(YamlFileSource::String(old_config_contents.into()))
```

Comment:
The original comment about intentionally using `String(old_config_contents)` instead of `File(old_config_file)` should be retained.

Reply:
Yes, will fix.

Reply:
d33a376c61bca7f28a7038ca22371fb142062f7d